diagnostic.md

```diff
@@ -3,53 +3,50 @@
 Using `inspect element`, write four selectors found on the main page for this
 GitHub repo.
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+div id "js-pjax-loader-bar"
+class "pjax-loader-bar"
+div id "start-of-content"
+div id "ajax-error-message"
 
 ## Question 2
 
 Using the diagram in the `README` file, what are two ways you could return ALL
 list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
-<!-- your answer starts here -->
+console.log( $( "li" ).get() );
```

Comment:
This works but the call to `console.log` is unnecessary.  We also want a second way of doing this.  From `<ol>`, two possible ways to select `<li>` elements are `$('ol > li');` and `$('ol').children();`.  Some other possible ways are `$('li');`, `$('body').find('li');`, and `$('body').children('ol').children('li');`.